### ISSUE TITLE
Reload Items At Index Paths

### ISSUE THREAD

**Issue by kolinkrewinkel** [kolinkrewinkel/KKGridView#19]
via -reloadItemsAtIndexPaths:  This should be a piece of cake with a for loop or enumeration and calling -setNeedsDisplay.

**Comment by jonsterling** [comment]
I'm working on this right now. I haven't been that active thus so far, so I want to be sure that I'm not implementing this incorrectly. What I've got is:

``` objc
- (void)reloadItemsAtIndexPaths:(NSArray *)indexPaths
{
    for (KKIndexPath *path in indexPaths) {
        KKGridViewCell *cell = [_visibleCells objectForKey:path];
        if (cell) {
            [cell removeFromSuperview];
            [_visibleCells removeObjectForKey:path];
        }

        cell = [_dataSource gridView:self cellForRowAtIndexPath:path];
        [_visibleCells setObject:cell forKey:path];
        cell.frame = [self rectForCellAtIndexPath:path];

        [self addSubview:cell];
        [self sendSubviewToBack:cell];
    }
}
```

Is that vaguely what you were expecting, or am I barking up the wrong tree?

**Comment by kolinkrewinkel** [comment]
I think that's a good idea, but we need to fix issue #14 first.  Want to move over to that one?

**Comment by jonsterling** [comment]
Sure, I'll take a look.
